infrastructure/environments/non-prod/main.tf

```diff
@@ -19,6 +20,11 @@ terraform {
   }
 }
 
+locals {
+  namespace        = "mhraproductsnonprod"
```

Comment:
should this use `var.ENVIRONMENT` too?

Reply:
Yeah, I think ideally it would be but `var.ENVIRONMENT` was already set to `non-prod` with a dash and the namespace without a dash and I don't want to change the names of things in non-prod and force them to be recreated so I've just stuck to the naming that's already there for the time being